packages/chrome-extension/package.json

```diff
@@ -57,7 +57,7 @@
     "@types/react-dom": "*",
     "@types/webextension-polyfill": "^0.10.7",
     "tsup": "*",
-    "type-fest": "^4.8.3",
+    "type-fest": "^0.21.0",
```

Comment:
@LekoArts any idea what's up with this one? 🤔

Reply:
No idea 😮